br/pkg/trace/tracing_test.go

```diff
@@ -10,37 +10,29 @@ import (
 	"time"
 
 	"github.com/opentracing/opentracing-go"
-	. "github.com/pingcap/check"
+	"github.com/stretchr/testify/require"
 )
 
-type testTracingSuite struct{}
-
-var _ = Suite(&testTracingSuite{})
-
-func TestT(t *testing.T) {
-	TestingT(t)
-}
-
 func jobA(ctx context.Context) {
 	if span := opentracing.SpanFromContext(ctx); span != nil && span.Tracer() != nil {
 		span1 := span.Tracer().StartSpan("jobA", opentracing.ChildOf(span.Context()))
 		defer span1.Finish()
 		ctx = opentracing.ContextWithSpan(ctx, span1)
 	}
 	jobB(ctx)
-	time.Sleep(time.Second)
+	time.Sleep(10 * time.Millisecond)
```

Comment:
Is this for accelerating tests?

Reply:
yes

Reply:
@shadw3002 can you elaborate why this change won't decrease stability?

Reply:
I think if the error of Sleep() is less than 5 ms, this test will work, but I am not sure about its stability.

Reply:
I think the sleep duration can not affect the result of the test.🤔

Reply:
@mmyj yes it should not. I totally agree that we should write determinate test - just check that this change won't introduce new unstable test that blocks every PR.

Reply:
I'm ok to accept the change and keep an eye on it if it causes unstable case.

Reply:
maybe I could call `/run-check_dev_2` over again and again to check its stability.